accepted/future-releases/spread-collections/implementation-plan.md

```diff
@@ -0,0 +1,145 @@
+# Implementation Plan for "Spread Collections"
+
+Owner: rnystrom@google.com ([@munificent](https://github.com/munificent/) on GitHub)
+
+Relevant links:
+
+* [Tracking issue](https://github.com/dart-lang/language/issues/47)
+* [Proposal](https://github.com/dart-lang/language/blob/master/accepted/future-releases/spread-collections/feature-specification.md)
+
+## Phase 0 (Prerequisite)
+
+### Experiment support
+
+Tools add support for experimental flags, if not yet supported. Tools refuse to
+run if supplied with unknown experimental flags.
+
+Projects embedded in other tools, like the Common Front-end (CFE) embedded in
+the VM or dart2js, must be able to receive experimental flags programmatically
+from the embedder, which can then accept them on the command line.
+
+### "spread-collections" Experimental flag
+
+The implementation of this feature should be hidden behind an [experiment
+flag][]. Tools must be passed the flag `--enable-experiment=spread-collections`
+to enable the feature.
+
+[experiment flag]: https://github.com/dart-lang/sdk/blob/master/docs/process/experimental-flags.md
+
+While this feature is under development, individual tools may have incomplete or
+changing implementations behind the flag. When all tools have completely
+implemented the feature, the the feature will be enabled by default, and the
+flag removed in a stable release.
+
+### Tests
+
+The language team adds tests for the feature.
+
+## Phase 1 (Foundation)
+
+### CFE
+
+The CFE implements parsing the new syntax, type checking it, and compiling it to
+Kernel. Since the CFE will be implementing constant evaluation, it also
+implements evaluating `...` in constant collections.
+
+This feature can likely be implemented entirely in the front end, so back-end
+support may not be needed. If it does require Kernel changes, the back end will
+need to handle those changes.
+
+### Analyzer
+
+The analyzer implements parsing the new syntax, type checking it, and
+evaluating `...` in constant collections.
+
+## Phase 2 (Tool Implementation)
```

Comment:
Add dartdoc.

Reply:
Is there anything dartdoc needs to do for this? It doesn't affect the signatures of anything.

Reply:
It affects that set of AST nodes it can see in constants, so I think it needs to be prepared to handle it or it may break.